README.md

```diff
@@ -1,52 +1,124 @@
+    <li><a href="#contact">Contact</a></li>
+  </ol>
+</details>
+
+
+
+<!-- ABOUT THE PROJECT -->
+## About The Project
+
+Our goal is to create a CMS for everyone in an organization. It's our belief that a CMS should inspire every one to express themselves on the web by providing a modern and flexible developer experience and an intuitive interface for content editors. ApostropheCMS is built for teams, and we want to make your job more fun.
+
+#### Built With
+
+* [Node](https://nodejs.org/en/)
+* [Nunjucks](https://mozilla.github.io/nunjucks/)
+* [MongoDB](https://www.mongodb.com/)
+
+<!-- GETTING STARTED -->
 ## Getting Started
 
-The [ApostropheCMS documentation](https://docs.apostrophecms.org/apostrophe/) will walk you through some simple steps to get your [local environment set up](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/setting-up-your-environment) with the technology stack that runs ApostropheCMS (Node.js, MongoDB, ImageMagick) along with installing ApostropheCMS itself.
+To get started with ApostropheCMS, follow these steps to set up a local development environment.
+
+#### Prerequisites
+
+We recommend installing the following with [Homebrew](https://brew.sh/) on macOS. If you're on Linux, you should use your package manager (apt or yum). If you're on Windows, we recommend the Windows Subsystem for Linux.
+
+| Software | Minimum Version | Notes
+| ------------- | ------------- | -----
+| Git  | Any
+| Xcode  | Current | Only needed on macOS
+| Node.js | 8.x | Or better
+| npm  | 6.x  | Or better
+| MongoDB  | 3.6  | Or better
+| Imagemagick (optional)  | Any | GIF support, faster image uploads
 
-Then [create a new project](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/creating-your-first-project) on your computer and work on building your beautiful website.
 
-The ApostropheCMS documentation will walk you through [editing page templates](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/editing-page-templates), [global content](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/global), [creating custom widgets](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/custom-widgets) and [navigation](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/building-navigation), along with [configuring reusable "pieces"](https://docs.apostrophecms.org/apostrophe/tutorials/getting-started/reusable-content-with-pieces) and [global settings](https://docs.apostrophecms.org/apostrophe/core-concepts/global-settings/settings). Plus, a whole lot more!
+#### New Project
+Our recomended way to start a new project with ApostropheCMS is to use the [Apostrophe CLI](https://github.com/apostrophecms/apostrophe-cli). Alternatively, you can simply clone this repo and `npm install`.
+##### Install the Apostrophe CLI
+ ```sh
+  npm install -g apostrophe-cli
+ ```
+##### Create a project
+ ```sh
+ apos create-project <shortname-without-spaces>
+ ```
```

Comment:
This is going to leave them still not knowing how to add a user, or log in, and if they do log in, they can't do anything interesting.

I am concerned that the more we add to the README, the more it feels like the complete docs of an unfinished project, rather than an obvious pointer to go read the real documentation and get started properly and not be frustrated / disappointed in what you get.

Maybe there's a way to make that segue at this point in the README?

Reply:
We could stop here, but then [link to the tutorial](https://docs.apostrophecms.org/getting-started/creating-your-first-project.html#generating-a-test-project). Replace `<shortname-without-spaces>` with `my-project`. Devs see that enough to know what it means. The tutorial will take it from there. Then the next paragraph about the shortname disappears.

Reply:
I was today years old when I learned we install this `apos` shorthand command too.

Yes I think we should guide people clearly to the tutorial. Or else finish the job to the point where something really cool happens and then guide them to the tutorial. "Let's try this cool CMS! Uh. A blank page and I can't log in? Bye Felicia"

Reply:
Agreed. I think people should be able to get-it-running without leaping to the docs, so if that includes creating a user, then it's trivial to add it here. Directing to the docs after is a great next step.